### PR TITLE
Animated flow due to data transfer

### PR DESCRIPTION
**Pull request by ambuznego**
image

### CODE REVIEW

File: assets/js/components/flows/FlowsIndex.jsx [hunk 1]
```diff
@@ -25,7 +26,10 @@ class FlowsIndex extends Component {
   };
 
   submitChanges = (newElementsMap) => {
-    const [completePaths, elementPositions] = getCompleteFlows(newElementsMap);
+    const [completePaths, elementPositions] = getCompleteFlows(
+      newElementsMap,
+      this.props.allResourcesQuery.data
```
Comment (vicmgs) [inline assets/js/components/flows/FlowsIndex.jsx:31]: doesnt look like this is used

File: assets/js/components/flows/FlowsIndex.jsx [hunk 2]
```diff
@@ -125,14 +172,24 @@ class FlowsIndex extends Component {
...
+            }}
           />
         </ReactFlowProvider>
       </DashboardLayout>
     );
   }
 }
 
-const getCompleteFlows = (newElementsMap) => {
+const getCompleteFlows = (newElementsMap, data) => {
```
Comment (vicmgs) [inline assets/js/components/flows/FlowsIndex.jsx:192]: second arg not needed?

File: assets/js/components/flows/FlowsIndex.jsx [hunk 3]
```diff
@@ -315,7 +389,18 @@ const generateInitialElementsMap = (data, flowPositions) => {
...
+          flowPositions.edges,
+          edge.source
+        );
+        const element = {
+          id,
+          source: edge.source,
+          target: edge.target,
+          animated: isAnimated,
+        };
+        console.log(element);
```
Comment (vicmgs) [inline assets/js/components/flows/FlowsIndex.jsx:403]: log

File: assets/js/components/flows/FlowsIndex.jsx (hunk 1 above)
Comment (ambuznego) [reply assets/js/components/flows/FlowsIndex.jsx]: you righttt

File: assets/js/components/flows/FlowsIndex.jsx (hunk 2 above)
Comment (ambuznego) [reply assets/js/components/flows/FlowsIndex.jsx]: ty for catching

File: assets/js/components/flows/FlowsIndex.jsx (hunk 3 above)
Comment (ambuznego) [reply assets/js/components/flows/FlowsIndex.jsx]: removed

**Review by vicmgs — APPROVED** [review]

**Pull request merged**